File: T2.1 SOAP_DOTNET_EUREKA_BANK_GR08/01.SERVIDOR/ws_eureka_bank_soap_dotnet/ec.edu.monster.model/CuentaModel.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace ws_eureka_bank_soap_dotnet.ec.edu.monster.model
{
    [Serializable]
    public class CuentaModel
    {
        public string ChrCuenCodigo { get; set; }       // Código de la cuenta
    public string ChrMoneCodigo { get; set; }       // Código de moneda
    public string ChrSucucodigo { get; set; }       // Código de sucursal
    public string ChrEmplCreaCuenta { get; set; }   // Código de empleado que crea la cuenta
    public string ChrClieCodigo { get; set; }       // Código de cliente
    public double DecCuenSaldo { get; set; }        // Saldo de la cuenta
    public DateTime DttCuenFechaCreacion { get; set; } // Fecha de creación
    public string VchCuenEstado { get; set; }       // Estado (ACTIVO, ANULADO, CANCELADO)
    public int IntCuenContMov { get; set; }         // Contador de movimientos
    public string ChrCuenClave { get; set; }        // Clave de la cuenta
 }
}

File: TI2.1  SOAP_JAVA_EUREKA_BANK_GR08/01.SERVIDOR/ws_eureka_bank_soap_java/src/main/java/ec/edu/monster/modelo/CuentaModel.java
package ec.edu.monster.modelo;

import java.sql.Date;

public class CuentaModel {
    
    // Tabla Cuenta DB
    private String chrCuenCodigo;       // Código de la cuenta
    private String chrMoneCodigo;       // Código de moneda
    private String chrSucucodigo;       // Código de sucursal
    private String chrEmplCreaCuenta;   // Código de empleado que crea la cuenta
    private String chrClieCodigo;       // Código de cliente
    private double decCuenSaldo;        // Saldo de la cuenta
    private Date dttCuenFechaCreacion;  // Fecha de creación de la cuenta
    private String vchCuenEstado;       // Estado de la cuenta (ACTIVO, ANULADO, CANCELADO)
    private int intCuenContMov;         // Contador de movimientos de la cuenta
    private String chrCuenClave;        // Clave de la cuenta

  
    
    public String getChrCuenCodigo() {
        return chrCuenCodigo;
    }

    public void setChrCuenCodigo(String chrCuenCodigo) {
        this.chrCuenCodigo = chrCuenCodigo;
    }

    public String getChrMoneCodigo() {
        return chrMoneCodigo;
    }

    public void setChrMoneCodigo(String chrMoneCodigo) {
        this.chrMoneCodigo = chrMoneCodigo;
    }

    public String getChrSucucodigo() {
        return chrSucucodigo;
    }

    public void setChrSucucodigo(String chrSucucodigo) {
        this.chrSucucodigo = chrSucucodigo;
    }

    public String getChrEmplCreaCuenta() {
        return chrEmplCreaCuenta;
    }

    public void setChrEmplCreaCuenta(String chrEmplCreaCuenta) {
        this.chrEmplCreaCuenta = chrEmplCreaCuenta;
    }

    public String getChrClieCodigo() {
        return chrClieCodigo;
    }

    public void setChrClieCodigo(String chrClieCodigo) {
        this.chrClieCodigo = chrClieCodigo;
    }

    public double getDecCuenSaldo() {
        return decCuenSaldo;
    }

    public void setDecCuenSaldo(double decCuenSaldo) {
        this.decCuenSaldo = decCuenSaldo;
    }

    public Date getDttCuenFechaCreacion() { 
        return dttCuenFechaCreacion;
    }

    public void setDttCuenFechaCreacion(Date dttCuenFechaCreacion) {
        this.dttCuenFechaCreacion = dttCuenFechaCreacion;
    }

    public String getVchCuenEstado() {
        return vchCuenEstado;
    }

    public void setVchCuenEstado(String vchCuenEstado) {
        this.vchCuenEstado = vchCuenEstado;
    }

    public int getIntCuenContMov() {
        return intCuenContMov;
    }

    public void setIntCuenContMov(int intCuenContMov) {
        this.intCuenContMov = intCuenContMov;
    }

    public String getChrCuenClave() {
        return chrCuenClave;
    }

    public void setChrCuenClave(String chrCuenClave) {
        this.chrCuenClave = chrCuenClave;
    }
}
